src/Payum/Stripe/StripeCheckoutGatewayFactory.php

```diff
@@ -16,7 +16,7 @@ class StripeCheckoutGatewayFactory extends GatewayFactory
      */
     protected function populateConfig(ArrayObject $config)
     {
-        if (!class_exists('Stripe')) {
+        if (!class_exists('\Stripe\Stripe')) {
             throw new \LogicException('You must install "stripe/stripe-php" library.');
```

Comment:
I suppose so, they seem to respect semver. Not sure if it's better to be cautious and only support up to the current version.